tools/toolchains/arm.py

```diff
@@ -37,6 +37,11 @@
     "please visit https://os.mbed.com/docs/mbed-os/latest/tools/index.html"
 )
 
+UARM_TOOLCHAIN_WARNING = (
+    "Warning: We noticed that you are using Micro Arm Toolchain. "
+    "We are deprecating the use of Micro Arm Toolchain soon. "
+    "Please upgrade your environment to Arm Compiler 6 and use default_lib to choose microlib "        
```

Comment:
The comment about "use default_lib to choose microlib" should be removed. Instead you should link a page that explains how to build the bare metal profile.

Reply:
I removed and added the link to bare-metal